src/conf.py

```diff
@@ -285,3 +311,11 @@
 
 # If true, do not generate a @detailmenu in the "Top" node's menu.
 #texinfo_no_detailmenu = False
+
+# add breathe support
+
+breathe_projects = { "axomdoc": "@CMAKE_CURRENT_BINARY_DIR@/../doxygen/xml/" }
```

Comment:
I think this should be built up the way that `html_static_path` is.  @white238 , can you confirm this?

Reply:
You can't use any CMake variables in conf.py.  It has to be a conf.py.in (which don't work with RTD).

Reply:
thanks.